bitnami/rabbitmq/templates/statefulset.yaml

```diff
@@ -100,6 +100,9 @@ spec:
                {{- if and .Values.persistence.enabled .Values.forceBoot.enabled }}
                if [ -d "{{ .Values.persistence.path }}/mnesia/${RABBITMQ_NODENAME}" ]; then rabbitmqctl force_boot; fi
                {{- end }}
+               {{- if .Values.rabbitmq.communityPlugins }}
+               ./opt/scripts/install-plugins.sh
```

Comment:
did you wanted to use
```
. /opt/scripts/install-plugins.sh
```
?

Reply:
I just wan to execute that sh file. `./opt/scripts/install-plugins.sh` does the job